manager/controllers/app/blueprint_controller.go

```diff
@@ -193,10 +199,9 @@ func (r *BlueprintReconciler) obtainSecrets(ctx context.Context, log *zerolog.Lo
 	return registrySuccessfulLogin, nil
 }
 
-func (r *BlueprintReconciler) applyChartResource(ctx context.Context, log *zerolog.Logger, chartSpec fapp.ChartSpec,
-	args map[string]interface{}, blueprint *fapp.Blueprint, releaseName string) (ctrl.Result, error) {
+func (r *BlueprintReconciler) applyChartResource(ctx context.Context, cfg *action.Configuration, chartSpec fapp.ChartSpec,
+	args map[string]interface{}, releaseNameSpace, releaseName string, log *zerolog.Logger) (ctrl.Result, error) {
```

Comment:
releaseNameSpace -> releaseNamespace
Can't it be extracted from the config?

Reply:
- releaseNameSpace -> releaseNamespace 
OK
- Can't it be extracted from the config? 
yes and no. It is stored in one of implementations of `kube.Interface` when it is Kubernetes.Client, it's available there, but there are 2 other implementations...